### PR TITLE
Release/1.2.0

### PR DESCRIPTION
**Pull request by luiskuhn**
**Description of changes**
Updated UI and added a vaadin window for the image viewer.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -72,10 +72,9 @@
 		<dependency>
 			<groupId>life.qbic</groupId>
 			<artifactId>omero-client-lib</artifactId>
-			<version>1.2.0</version>
+			<version>1.2.1-SNAPSHOT</version>
```
Comment (wow-such-code) [inline pom.xml:75]: you need to release this dependency first, no snapshots are allowed

Comment (luiskuhn) [reply pom.xml:78]: ok, then I need to get 1.2.1 released

Comment (luiskuhn) [reply pom.xml]: done

**Review by wow-such-code — APPROVED** [review]
👍

**Pull request merged**